packages/react-native/src/primitives/Radio/Radio.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useCallback, useMemo } from 'react';
+import { Pressable, View, ViewStyle } from 'react-native';
+
+import { Label } from '../index';
+
+import { RadioProps } from './types';
+import { styles } from './styles';
+
+export default function Radio<T>({
+  buttonStyle,
+  disabled,
+  label,
+  labelPosition = 'end',
+  labelStyle,
+  onChange,
+  selected,
```

Comment:
Just a reminder that we need to make sure we have logic in place to handle both controlled and uncontrolled use cases ([React docs](https://reactjs.org/docs/uncontrolled-components.html))

Reply:
As in [something like this](https://github.com/aws-amplify/amplify-ui/blob/in-app-messaging/main/packages/react-native/src/primitives/Checkbox/Checkbox.tsx#L30)?